src/tools/mdbm_config.cc

```diff
@@ -42,7 +42,6 @@ static uint SYS_PAGESIZE;    // system page size
 static uint CONFIG_MAXPAGE = 1 << 24;   // 16MB (subtract system page size later)
 static const int MBYT = 1024 * 1024;
 static const uint ENTRY_OVERHEAD  =   8;     // MDBM per-entry overhead is 8 bytes
-static const uint PAGE_OVERHEAD   =  16;     // MDBM per-page overhead
```

Comment:
Thanks.
It didn't, I also didn't see any docs that related to this, nor did I see anything that looked like it seemed like it would be using this.